harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
does this test only deal with own properties (it seems like it due to the `delete`)? if so, can this entire test not simply `return !desc || !desc.configurable;`?